src/VoteView.js

```diff
@@ -2,25 +2,51 @@ import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import axios from 'axios';
 import CandidateVotes from './Candidate';
+import { Link } from 'react-router-dom';
+import { Button } from './Candidate';
+
 const url =
   'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/candidates';
 
-function VoteView() {
+function LoginoutButton({ isLogined, handleLogoutButton }) {
+  if (isLogined) {
+    return (
+      <UserButton onClick={handleLogoutButton} r={128} g={203} b={196}>
```

Comment:
`AuthActionButton`이 더 좋았을듯!

```suggestion
      <AuthActionButton onClick={handleLogoutButton} r={128} g={203} b={196}>
```